dist/Makefile.tmpl

```diff
@@ -144,8 +144,6 @@ lib_gen/Lib_RandomBuffer_System_gen.exe: lib/Lib_RandomBuffer_System_bindings.cm
 # https://www.gnu.org/software/make/manual/html_node/Canceling-Rules.html
 %.exe:
 
-all: ocamlevercrypt.cmxa
```

Comment:
why remove this?

Reply:
This was redundant, there's an earlier `all: libevercrypt.$(SO)`

Reply:
I don't understand,building libevercrypt.$(SO) doesn't imply building `ocamlevercrypt.cmxa`, does it?

Reply:
No, but I don't think this changes anything, this target was basically dead, right? `ocamlevercrypt.cmxa` is built by `install-hacl-star-raw`.

Reply:
Fair enough, but can that work is the Makefile is shared but we're only building the OCaml library in gcc-compatible?

Reply:
there's a guard here: 

103 ifneq (,$(wildcard lib_gen))